### PR TITLE
Update utils.py

### PR DESCRIPTION
**Pull request by SaoYan**
Prevent the "divided by zero" error.

### CODE REVIEW

File: torchvision/utils.py [hunk 1]
```diff
@@ -51,7 +51,8 @@ def make_grid(tensor, nrow=8, padding=2,
 
         def norm_ip(img, min, max):
             img.clamp_(min=min, max=max)
-            img.add_(-min).div_(max - min)
+            if min != max:
+                img.add_(-min).div_(max - min)
```
Comment (alykhantejani) [inline torchvision/utils.py]: I think we still want to have the `img.add_(-min)` just avoid the div by 0 if `min==max`

Comment (vishwakftw) [reply torchvision/utils.py]: We could have something as simple as `img.add_(-min).div_(max - min + 1e-10)` (or any suitable small number).

Comment (fmassa) [reply torchvision/utils.py]: Both options are fine for me

**Comment by vishwakftw** [comment]
I believe that this issue has been addressed in a recent PR, so can this be closed?

**Pull request closed without merging**